askama_derive/src/generator.rs

```diff
@@ -15,6 +15,14 @@ use parser::node::{
 use parser::{Expr, Node};
 use quote::quote;
 
+fn insert_variable_at_the_end(args: &mut Vec<Expr<'_>>, var_name: String) {
```

Comment:
You should know better about where to put this by now. 😛

Reply:
Oh no. :o I was trying to remember if it was "a la C" (ie, defined before used) or the opposite, picked the wrong one obviously. ^^'

Reply:
It says `struct Generator` like 5 lines below, which should be a decent indication?

Reply:
Sorry, I don't understand what you mean.